src/components/Footer/FooterLinks/index.jsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import boleto from './icons/boleto.svg';
+import elo from './icons/elo.svg';
+import mastercard from './icons/mastercard.svg';
+import paypal from './icons/paypal.svg';
+import visa from './icons/visa.svg';
+import telephone from './icons/telephone.svg';
+import email from './icons/email.svg';
+import {
+    FooterContainer,
+    FooterCategories,
+    CategoryLinks,
+    SupportInfo,
+    FooterImages,
+} from './styles';
+
+const FooterLinks = [
+    {
+        title: 'Categories',
+        links: [
+            {
+                title: 'Peças',
+                path: 'xxx',
+            },
+            {
+                title: 'Serviços',
+                path: 'xxx',
+            },
+            {
+                title: 'Manutenção',
+                path: 'xxx',
+            },
+            {
+                title: 'Sistema de Trocas',
+                path: 'xxx',
+            },
+        ],
+    },
+    {
+        title: 'Institucional',
+        links: [
+            {
+                title: 'Quem Somos',
+                path: 'xxx',
+            },
+            {
+                title: 'Nosso Historia',
+                path: 'xxx',
+            },
+            {
+                title: 'Trabalhe Conosco',
+                path: 'xxx',
+            },
+        ],
+    },
+    {
+        title: 'Ajuda',
+        links: [
+            {
+                title: 'Política de Privacidade',
+                path: 'xxx',
+            },
+            {
+                title: 'Política de Troca',
+                path: 'xxx',
+            },
+            {
+                title: 'Termos e Condições',
+                path: 'xxx',
+            },
+            {
+                title: 'FAQ',
+                path: 'xxx',
+            },
+            {
+                title: 'Fale Conosco',
+                path: 'xxx',
+            },
+        ],
+    },
+    {
+        title: 'Suporte',
+        links: [
+            {
+                number: '080000000',
+                email: 'ajuda@scania.com.br',
+                emailicon: [<FooterImages src={email} height="20" width="20" />],
+                numbericon: [<FooterImages src={telephone} height="20" width="20" />],
+                img: [
+                    <FooterImages width="30" height="30" src={boleto} />,
+                    <FooterImages width="30" height="30" src={elo} />,
+                    <FooterImages width="30" height="30" src={mastercard} />,
+                    <FooterImages width="30" height="30" src={paypal} />,
+                    <FooterImages width="30" height="30" src={visa} />,
+                ],
+            },
+        ],
+    },
+];
+
+const MainFooter = () => {
+    return (
+        <FooterContainer>
+            {FooterLinks &&
+                FooterLinks.map((FooterLinks, index) => {
```

Comment:
Note that you are using a variable with the same name here. I suggest applying the follow instruction to avoid this.

```suggestion
                FooterLinks.map((footerLink, index) => {
```